src/CLICommand/cli_command.ts

```diff
@@ -13,7 +16,14 @@ export interface CommandDescriptor {
 }
 
 const programAsUnknown: unknown = program;
-const programApi: CliApiObject = programAsUnknown as CliApiObject;
+export const programApi: CliApiObject = programAsUnknown as CliApiObject;
+
+programApi.name('ardrive');
+programApi.version(CLI_APP_VERSION);
+programApi.addHelpCommand(true);
+programApi.usage('[command] [command-specific options]');
+// Override the commander's default exit (process.exit()) to avoid abruptly interrupting the script execution
+programApi.exitOverride();
```

Comment:
Why did these assignments get lifted up from the constructor into this module's scope?

Reply:
You've answered here: https://github.com/ardriveapp/ardrive-cli/pull/293#issuecomment-1180862266 

:)

Reply:
> I've made some changes to the CLICommand implementation because there was a node warning talking about a "possible listeners memory leak detected". https://stackoverflow.com/questions/9768444/possible-eventemitter-memory-leak-detected
> The listeners are now set only once. See: [2208aac](https://github.com/ardriveapp/ardrive-cli/pull/293/commits/2208aac4f144366cd396018b319edcfe36376a1c)

Because of a warning. The behavior is kept because the tests are running.

Reply:
This was actually how I originally coded it until the testing issue came up, at which point I moved everything back to the CLICommand constructor. This make much more logical sense - to set the program up 1 time.